tests/test_data.py

```diff
@@ -1 +1,10 @@
+from flagging_site.data.hobolink import get_hobolink_data
+from datetime import datetime
+import pandas as pd
+
+def test_get_hobolink_data():
+    df = get_hobolink_data('code_for_boston_export')
+    last_timestamp = df['time'].iloc[-1]
+    time_difference = (datetime.now()-last_timestamp)
```

Comment:
You don't even need to import the base `datetime` module. You can use `pd.to_datetime('today')`, which returns the local current time. I typically recommend this because the base datetime module in Python is pretty awful, and Pandas's way of handling datetimes is a bit nicer.

Also, I'd put spaces between the minus sign; that change makes it aesthetically nicer, and you have plenty of horizontal real estate to work with so don't be shy.

Reply:
Updated